### DISCUSSION TITLE
PR/PO Equipment mic shelf

### DISCUSSION THREAD

**Discussion by Cheng-ChiAn** [J-tour/J-tour.github.io#26] [General]
### 麥克風架採購草案
下列多種價位麥架，預計買兩個。若是網購，運費100上下。請留下意見，偏好哪種功能或價位，之後收集完意見即提採購案。

1. 貴的
- 優點：穩重 可能較耐用 
- 缺點：貴

![Image](https://github.com/J-tour/tsaegow/assets/163726368/aa7dee82-62c4-46e6-9595-edcbdffffb90)

2. 沒那麼貴的
> 有黑色跟銀色，黑色貴50塊

- 優點：台灣製造，本體算夠堅固堪用
- 缺點：質感沒貴得好，附的麥夾品質偏爛
> 我有一個麥夾已經爆了

![Image](https://github.com/J-tour/tsaegow/assets/163726368/41011295-214a-43a5-b377-1b7c4ec97bee)

3. 神奇便宜的
- 優點：便宜
- 缺點：可能不夠耐用

![Image](https://github.com/J-tour/tsaegow/assets/163726368/ce574a7e-fae3-40c9-b95c-f5c8413f54b5)

4. 有複合功能的
- 優點：可放平板
- 缺點：最貴 品項不多

![Image](https://github.com/J-tour/tsaegow/assets/163726368/3f79af28-58ff-4882-a97e-be23a1dd227d)

**Comment by Sean880128** [comment]
# Sean的想法 
* 價格中(1200)~高(2100)
* 耐用正常保管至少可以使用5年~10年 
* 麥夾子真的要夠穩

我偏好  1. 昂貴 及 4. 複合式
順便說一下我們前幾周去新竹樂器行看的價格
一般的落在680上下 (社辦再用的類型)
昂貴的落在2100上下 (黑色簡單耐用的感覺)

**Reply by ShengHaoLo** [reply]
建議附上供應商比較，以比價最佳利益

**Comment by ShengHaoLo** [comment]
### FQA
- 麥夾子是否是統一規格？

**Reply by ShengHaoLo** [reply]
目前[昊昊]( @ShengHaoLo )有一組錄音室用麥架含手機支架可以暫時使用，僅須再購買一隻手機支架放置於另一個麥架即可。

**Reply by ShengHaoLo** [reply]
由於上次中原文創園區表演時已送走昊昊的麥夾，建議再行添購，可參照麒安近期列的計畫安排

**Reply by ShengHaoLo** [reply]
@pighome7 提供了兩隻手機架，再請新增資訊至設備清單 感謝

**Comment by xinyannn1** [comment]
我會選擇 1.昂貴 或 4.複合式，
以長久來看品質好一點的設備應能有較長使用壽命，
只不過複合式的可能要考慮它能承重多少、是否只能架手機...等等。

複合式參考，但這種的重量應該很重
https://shopee.tw/%E8%AD%9C%E6%9E%B6-%E9%BA%A5%E5%85%8B%E9%A2%A8%E6%9E%B6-%E5%A4%A7%E8%AD%9C%E6%9E%B6-%E6%96%9C%E6%9E%B6-%E7%9B%B4%E6%9E%B6-%E6%96%9C%E6%A1%BF-%E5%A5%97%E7%B5%84-(%E8%B4%88-%E9%BA%A5%E5%85%8B%E9%A2%A8%E5%A4%BE-)-%E7%B6%B2%E9%9F%B3%E6%A8%82%E5%9F%8E--i.5280812.140440245
